### ISSUE TITLE
Improve MotK behavior

### ISSUE THREAD

**Issue by OleksandrBerchenko** [seastan/ringsdb#175]
1. When any MotK hero is chosen, the MotK contract itself should be automatically added.
2. When any MotK hero is chosen, the corresponding ally should be automatically added.
3. Detect more than one MotK hero.

**Comment by baptistemanson** [comment]
- When any MotK hero is removed, the corresponding ally(ies) should be removed
- In case the ally(ies) are already added when adding an MotK nothing should happen.
Should I add the corresponding ally in the deck? I don't remember the rules about this.

**Comment by OleksandrBerchenko** [comment]
> When any MotK hero is removed, the corresponding ally(ies) should be removed
I don't think we should do anything like that, because the deck remains valid.

> In case the ally(ies) are already added when adding an MotK nothing should happen.
Yes, of course.
